pages/library/_viewId.vue

```diff
@@ -1,36 +1,34 @@
 <template>
   <div>
-    <h1>{{ Name }}</h1>
-    <v-list>
-      <v-list-item
+    <h1>
+      <span>{{ Name }}</span>
+    </h1>
+    <div class="cardsContainer">
+      <CardBuilder
         v-for="item in items"
         :key="item.Id"
         :to="`../itemdetails/${item.Id}`"
```

Comment:
```suggestion
        :to="`../item/${item.Id}`"
```

It's more natural as an URL, shorter to remember and we can use its children for stuff like playing or editing, which makes the URLs super predictable.